src/engine/QueryPlanner.cpp

```diff
@@ -499,7 +499,7 @@ bool QueryPlanner::checkUsePatternTrick(
 
       // check that all selected variables are outputs of
       // CountAvailablePredicates
```

Comment:
If you have an asterisk in the query,
just return false, and then you can always assume, that you have "selectVAriables"